appveyor.yml

```diff
@@ -1,5 +1,5 @@
 ---
-image: Visual Studio 2017
+image: Visual Studio 2019
```

Comment:
What about Visual Studio 2017? Does that not work, or did you simply change this because you were only able to test using VS2019?